plugins/finder/jevents/jevents.php

```diff
@@ -384,19 +342,121 @@ protected function getItem($id)
 	 */
 	protected function getStateQuery()
 	{
-		$sql = $this->db->getQuery(true);
+		$db     = $this->db;
+		$query  = $db->getQuery(true);
+
 		// Item ID
-		$sql->select('a.evdet_id as id');
+		$query->select('a.ev_id');
+
```

Comment:
We can't change from event detail id to event id - its a fundamental change in functionality of the search and would stop us from finding custom repeats